src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -149,13 +150,15 @@ class JesBackend extends Backend with LazyLogging {
     val filesWithinExpressions = Try(
       call.task.outputs.map {
         taskOutput => taskOutput.expression.preevaluateExpressionForFilenames(scopedLookupFunction, engineFunctions)
-      }.flatMap({_.get})
+      }.flatMap({
```

Comment:
That's IntelliJ's default style.  What hero will modify the IntelliJ style to save us from this menace?